block.go

```diff
@@ -16,20 +16,22 @@ import (
 type DriveType int
 
 const (
-	DRIVE_TYPE_UNKNOWN DriveType = iota
-	DRIVE_TYPE_HDD               // Hard disk drive
-	DRIVE_TYPE_FDD               // Floppy disk drive
-	DRIVE_TYPE_ODD               // Optical disk drive
-	DRIVE_TYPE_SSD               // Solid-state drive
+	DRIVE_TYPE_UNKNOWN     DriveType = iota
+	DRIVE_TYPE_HDD                   // Hard disk drive
+	DRIVE_TYPE_FDD                   // Floppy disk drive
+	DRIVE_TYPE_ODD                   // Optical disk drive
+	DRIVE_TYPE_SSD                   // Solid-state drive
+	DRIVE_TYPE_USB_STORAGE           // USB flash drive, SSD/HDD by USB
```

Comment:
I'm uncomfortable adding a "USB_STORAGE" drive type. I don't believe that USB is actually a drive type; rather, USB is a bus and set of storage/transfer protocol standards. The drive type is either HDD, SSD or ODD (since there are some CD/DVD optical drives that are connected via USB).